### DISCUSSION TITLE
现在有一个文件操作，比如说读取data.txt中字母A的数量，并把A的数量输出到屏幕

### DISCUSSION THREAD

**Discussion by QuincyWuzefeng** [criyle/go-judge#117] [Q&A] [answered]
代码如下
`#include <stdio.h>

int main() {
    // 打开输入文件
    FILE* input_file = fopen("data.txt", "r");
    if (input_file == NULL) {
        printf("无法打开输入文件\n");
        return 1;
    }
    // 统计字符 'A' 的数量
    int count = 0;
    int ch;
    while ((ch = fgetc(input_file)) != EOF) {
        if (ch == 'A' || ch == 'a') {
            count++;
        }
    }
    // 将结果写入输出文件
    printf("输出结果%d\n", count);
    // 关闭文件
    fclose(input_file);

    return 0;
}
`
调用的命令行为：
{"cmd":[{"args":["/w/main"],"env":["PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin","LANG=en_US.UTF-8","LC_ALL=en_US.UTF-8","LANGUAGE=en_US:en","HOME=/w"],"files":[{"name":"stdout","max":33554432},{"name":"stderr","max":16777216}],"cpuLimit":1200000000,"clockLimit":3600000000,"memoryLimit":373293056,"procLimit":128,"stackLimit":134217728,"copyIn":{"main":{"fileId":"WKPVTAG2"},"data.txt":{"content":"AABBAA\n"}},"copyOut":["stderr","stdout"]}]}
为什么结果没有输出到stdout中？命令哪部分有问题

**Comment by QuincyWuzefeng** [comment]
[{"status":0,"exitStatus":0,"time":1165614,"memory":221184,"runTime":5385604,"files":{"stderr":"输出结果4\n","stdout":""},"originalStatus":"Accepted"}]
屏幕输出的结果 输出到了stderr文件了

**Reply by criyle** [reply, marked as answer]
files 参数的下标是有意义的，0，1，2分别是 stdin stdout 和 stderr ，是linux标准。和 name 没有关系。

**Reply by QuincyWuzefeng** [reply]
好的 明白了，谢谢！
